lib/estella/query.rb

```diff
@@ -102,11 +98,20 @@ def term_search_fields
         .flatten
     end
 
-    def add_filters
-      if params[:indexed_fields]
-        params[:indexed_fields].each do |field, opts|
-          must term: { field => params[field] } if opts[:filter] && params[field]
-        end
+    def add_indexed_fields
```

Comment:
`add_filters` is still a more accurate name for this method - it adds _filters_ to the query, not just any indexed fields. the filters are sourced from the list of `indexed_fields` but we are not simply adding indexed fields to the query.